satellite/satellitedb/accounting.go

```diff
@@ -8,9 +8,11 @@ import (
 	"database/sql"
 	"time"
 
+	"github.com/skyrings/skyring-common/tools/uuid"
 	"github.com/zeebo/errs"
 
 	"storj.io/storj/pkg/accounting"
+	acct "storj.io/storj/pkg/accounting"
```

Comment:
Avoid using aliases, they give inconsistent names for packages and make harder to understand what is the shortened name.